template.tpl

```diff
@@ -1071,6 +1080,48 @@ ___TEMPLATE_PARAMETERS___
       }
     ]
   },
+  {
+    "type": "GROUP",
+    "name": "buttonClickTrackingConfig",
+    "displayName": "Filter",
+    "groupStyle": "ZIPPY_OPEN",
+    "subParams": [
+      {
+        "type": "SELECT",
+        "name": "buttonClickTrackingFilterChoice",
+        "displayName": "Filter Type",
+        "macrosInSelect": false,
+        "selectItems": [
+          {
+            "value": "allowlist",
+            "displayValue": "Allowlist"
```

Comment:
The display values could probably just be "Allow" and "Deny", but I don't think it's a big deal.

Reply:
Allow/Deny sounds a bit better to me too, updated.